### PR TITLE
Final Changes

### PR DESCRIPTION
**Pull request by iamharnad**
Added ARIA attributes
Modified Styles
Modified Products
Added different Types of Logo
Added Links in footer

### CODE REVIEW

**Review by spraveenofficial — APPROVED** [review]

**Review by srishti-maurya — APPROVED** [review]

File: cart.html [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
-    <title>all products | amazon</title>
+    <title>Add on Cart | Mike</title>
   </head>
   <body>
     <div class="container">
       <div class="navbar">
         <div class="logo">
           <a href="./index.html"
-            ><img src="./images/logo.png" alt="" width="125px"
+            ><img src="./images/logo-title-1.png" alt="site-logo" width="125px"
```
Comment (theVivek-kumar) [inline cart.html:30]: use rem

File: cart.html [hunk 2]
```diff
@@ -40,7 +40,11 @@
           </ul>
         </nav>
         <a href="./cart.html"
-          ><img src="./images/cart.png" alt="" width="30px" height="30px"
+          ><img
+            src="./images/cart.png"
+            alt="cart-button"
+            width="30px"
+            height="30px"
```
Comment (theVivek-kumar) [inline cart.html:47]: use rem

File: products.html [hunk 3]
```diff
@@ -40,11 +43,15 @@
...
         <a href="./cart.html"
-          ><img src="./images/cart.png" alt="" width="30px" height="30px"
+          ><img
+            src="./images/cart.png"
+            alt="cart-image"
+            width="30px"
+            height="30px"
         /></a>
         <img
           src="./images/menu.png"
```
Comment (theVivek-kumar) [inline products.html:53]: use rem here

Comment (iamharnad) [reply products.html:53]: modifed the requested changes

File: cart.html (hunk 2 above)
Comment (iamharnad) [reply cart.html:47]: modifed the requested changes

File: cart.html (hunk 1 above)
Comment (iamharnad) [reply cart.html:30]: modifed the requested changes

**Comment by Rishi05051997** [comment]
Looks Good

**Pull request merged**